modules/cluster-logging-updating-logging.adoc

```diff
@@ -148,6 +148,7 @@ green  open   infra-000008
 green  open   infra-000004                                                          rtDSzoqsSl6saisSK7Au1Q   3 1       226717            0        297            148
 green  open   infra-000012                                                          RSf_kUwDSR2xEuKRZMPqZQ   3 1       227623            0        295            147
 green  open   .kibana_7                                                             1SJdCqlZTPWlIAaOUd78yg   1 1            4            0          0              0
+// TBD - Update from `.operations` to `infra-` in the following example for https://bugzilla.redhat.com/show_bug.cgi?id=1873372
 green  open   .operations.2020.06.30                                                aOHMYOa3S_69NJFh2t3yrQ   3 1      4206118            0       8998           4499
```

Comment:
You can delete from the sample output lines that include `.project...` and `.operations...`. AFAICS, we already have `.infra-0000..` (corresponds to old `.operations...`) and `.app-0000*` (corresponds to old `.project...`).